bin/coq/coqtop.ml

```diff
@@ -114,11 +114,23 @@ let term =
               , "DuneExtraction" )
           in
           let* (_ : unit * Dep.Fact.t Dep.Map.t) =
-            let deps =
-              Dune_rules.Coq_rules.deps_of ~dir ~use_stdlib ~wrapper_name
+            let dep_map =
+              Dune_rules.Coq_rules.get_dep_map ~dir ~use_stdlib ~wrapper_name
                 coq_module
             in
-            Action_builder.run deps Eager
+            let vo_target =
+              Path.Build.set_extension ~ext:".vo"
```

Comment:
I think we should encapsulate this and instead have an API where we pass to the map a Coq_module.t